package.json

```diff
@@ -64,18 +66,17 @@
         "dom-align": "1.12.3",
         "lottie-web": "5.8.1",
         "memoize-one": "6.0.0",
+        "react": "17.0.2",
+        "react-dom": "17.0.2",
         "react-flip-toolkit": "7.0.13",
         "react-is": "18.1.0",
         "resize-observer-polyfill": "1.5.1",
         "scroll-into-view-if-needed": "2.2.25",
         "shallowequal": "1.1.0"
     },
     "peerDependencies": {
-        "@types/react": "17.0.39",
-        "@types/react-dom": "17.0.11",
-        "react": "17.0.2",
-        "react-app-polyfill": "3.0.0",
```

Comment:
This is seemingly only used by jest setupFiles... I've moved it to devDependencies. Its certainly not a peer dependency unless I'm missing something

Reply:
Should be okay.